apps/webcomponents/project.json

```diff
@@ -47,12 +47,6 @@
               "maximumError": "1mb"
             }
           ],
-          "fileReplacements": [
-            {
-              "replace": "apps/webcomponents/src/environments/environment.ts",
-              "with": "apps/webcomponents/src/environments/environment.prod.ts"
-            }
-          ],
```

Comment:
Indeed, as we do not have a real dev mode for the web components we can get rid of it entirely. Maybe something to set up in the future? Or do you think we won't be able to set up a different "dev env" than bootstrapping the web components as an app?

Reply:
I would like to do that in another PR yes